dist/Dockerfile

```diff
@@ -17,4 +17,6 @@ ADD entrypoint.sh /entrypoint.sh
 RUN chmod +x /entrypoint.sh
 RUN ls
 
+RUN apt-get update && apt-get install -y openssh-client
+
```

Comment:
This change needs to be done upstream. See https://github.com/game-ci/docker/pull/117
```suggestion
```